docs/fundamentals/code-analysis/style-rules/expression-level-preferences.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Expression-level preferences"
+description: "Learn about code analysis language rules for expression-level preferences"
+ms.date: 09/30/2020
+ms.topic: reference
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# Expression-level preferences
+
+## Overview
+
+The style rules in this section concern following expression-level preferences:
+
+- [Use object initializers](ide0017.md)
```

Comment:
```suggestion
- [Use object initializers (IDE0017)](ide0017.md)
```

I think adding the IDs helps the reader correlate what they see in the TOC, which is just the IDs. Also, I think we should sort these by ID so it matches the order in the TOC.